src/vehicle.cpp

```diff
@@ -5323,6 +5338,29 @@ void vehicle::gain_moves()
     }
 }
 
+bool vehicle::decrement_summon_timer()
+{
+    if( !summon_time_limit ) {
+        return false;
+    }
+    if( *summon_time_limit <= 0_turns ) {
+        for( auto &p : parts ) {
+            for( const auto &e : p.items ) {
+                g->m.add_item_or_charges( global_part_pos3( p ), e );
+            }
+            p.items.clear();
+        }
+        if( g->u.sees( global_pos3() ) ) {
+            add_msg( m_info, string_format( _( "Your %s winks out of existence." ), name ) );
```

Comment:
`add_msg` already does string formatting, you don't need to do this on your own.